<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Ploter" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>541</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>201</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Dimx</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="dimx">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Dimy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="dimy">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Dimz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="dimz">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_3"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Raio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_14"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Esfera</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Elipse</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="horizontalSliderRx">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Rx</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_4"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QSlider" name="horizontalSliderRy">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Ry</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_5"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QSlider" name="horizontalSliderRz">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Rz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_6"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Caixa</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QSlider" name="horizontalSlidercx">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>dimx</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_7"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QSlider" name="horizontalSlidercy">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>dimy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_8"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QSlider" name="horizontalSlidercz">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>dimz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_9"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Cor</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QSlider" name="horizontalSliderCorR">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>r</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_10"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QSlider" name="horizontalSliderCorG">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>g</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_11"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QSlider" name="horizontalSliderCorB">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>b</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_12"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>plano Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderPlanoz">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_13"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>320</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QPushButton" name="pushButtonCutvoxel">
       <property name="text">
        <string>CutVoxel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCutElipse">
       <property name="text">
        <string>CutElipse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCutEsfera">
       <property name="text">
        <string>CutEsfera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCutCaixa">
       <property name="text">
        <string>CutCaixa</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>320</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QPushButton" name="pushButtonPutVoxel">
       <property name="text">
        <string>PutVoxel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPutelipse">
       <property name="text">
        <string>PutElipse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPutEsfera">
       <property name="text">
        <string>PutEsfera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCaixa">
       <property name="text">
        <string>PutCaixa</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonSavar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMudaTamanho">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MudarTamanho</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ploter</class>
   <extends>QWidget</extends>
   <header location="global">ploter.h</header>
   <container>1</container>
   <slots>
    <slot>setAmplitude(int)</slot>
    <slot>mudaDimx(int)</slot>
    <slot>mudaDimy(int)</slot>
    <slot>mudaDimz(int)</slot>
    <slot>mudaRaio(int)</slot>
    <slot>mudaPutvoxel()</slot>
    <slot>mudaPutEsfera()</slot>
    <slot>mudaPutElipse()</slot>
    <slot>mudaPutCaixa()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dimx</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dimy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dimz</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dimx</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>mudaDimx(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dimy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>mudaDimy(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>mudaRaio(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPutVoxel</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>mudaPutvoxel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPutEsfera</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>mudaPutEsfera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPutelipse</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>mudaPutElipse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCaixa</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>mudaPutCaixa()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRx</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>778</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRz</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlidercx</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlidercy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>778</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlidercz</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderCorR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderCorG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_11</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderCorB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_12</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderPlanoz</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_13</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_14</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>785</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
